dds/DCPS/MultiTopicDataReader_T.cpp

```diff
@@ -108,11 +105,11 @@ MultiTopicDataReader_T<Sample, TypedDataReader>::join(
       ReturnCode_t ret = other_dri->read_instance_generic(other_data.ptr_,
         info, ih, READ_SAMPLE_STATE, ANY_VIEW_STATE, ALIVE_INSTANCE_STATE);
       if (ret != RETCODE_OK && ret != RETCODE_NO_DATA) {
-        std::ostringstream rc_ss;
-        rc_ss << ret;
-        throw std::runtime_error("In join(), incoming DataReader for " +
-          OPENDDS_STRING(other_topic) + " read_instance_generic, error #" +
-          rc_ss.str());
+        std::ostringstream ss;
```

Comment:
Now that int-to-string isn't used, this doesn't need ostringstream (also below).